### DISCUSSION TITLE
Minor misspelling

### DISCUSSION THREAD

**Discussion by Vulpine05** [skills/.github#424] [Release-based workflow]
In Step three, under the "Release branches and main" section, "repace" should be spelled as "replace".

Great course!  Thank you!

**Comment by cmwilson21** [comment]
@Vulpine05 Thanks for pointing this out! Would you mind opening an [issue](https://github.com/skills/release-based-workflow/issues/new) with this typo? You can even submit a PR with the fix if you are comfortable with that!

**Reply by Vulpine05** [reply]
@cmwilson21 , yeah I can do both of those.  I'll take care of that later tonight.

**Comment by cmwilson21** [comment]
Closing this discussion as the fix has been provided 💖
